### PR TITLE
Read import path from component json

### PR DESCRIPTION
**Pull request by broccolinisoup**
Having a new next entry point made the current auto generated component import path invalid for next components such as tooltip. Discussed this with @camertron and instead of generating the import path based on the component status, we think adding another `importPath` as a field on the component.docs.json and read the import path directly from field is less prune to invalid path generation. This PR alters how we form the import path.

Associated Primer React PR: https://github.com/primer/react/pull/4461

### CODE REVIEW

File: gatsby-node.esm.js [hunk 1]
```diff
@@ -378,6 +378,7 @@ exports.createPages = async ({actions, graphql}) => {
           name
           status
           a11yReviewed
+          importPath
```
Comment (broccolinisoup) [inline gatsby-node.esm.js:381]: @camertron I am not super sure about the graphql schema updates. Feel free to edit it on the PR or suggest changes if they are not correct.

Comment (camertron) [inline gatsby-node.esm.js:381]: We only need to add `importPath` here if we're using it to build pages, which it appears we're not. We should be able to remove this line and the other one in this file 👍 Not a blocker tho!

**Review by camertron — APPROVED** [review]
Nice!!

File: gatsby-node.esm.js (hunk 1 above)
Comment (broccolinisoup) [reply gatsby-node.esm.js:381]: Hmm, okay. Thanks! I'll remove it now.

Comment (broccolinisoup) [inline gatsby-node.esm.js:381]:
```suggestion
```

**Review by camertron — APPROVED** [review]

**Pull request merged**